README.md

```diff
@@ -92,6 +128,61 @@ Install the requirements mentioned above and copy the distribution into
 any directory you like.
 
 
+Execute
+-------
+
+```
+python2 ../ci-dnf-stack/cidnfstack.py build -c dnf-pull-requests tito
+```
+Localtest (omly for project that use tito):
+```
+python2 /path/to/cidnfstack.py build
+```
+Will test the project in current directory.
+
+Or:
+```
+python2 /path/to/cidnfstack.py build -l /path/to/project/1/ -l /path/to/project/2/
```

Comment:
usage: cidnfstack.py setup [-h] [--add-repository URL]
                           CHROOT [CHROOT ...] PROJECT

Create a new Copr project.

positional arguments:
  CHROOT                the chroots to be used in the project ("22" adds
                        "fedora-22-i386, fedora-22-x86_64", "23" adds
                        "fedora-23-i386, fedora-23-x86_64", "rawhide" adds
                        "fedora-rawhide-i386, fedora-rawhide-x86_64")
  PROJECT               the name of the project

optional arguments:
  -h, --help            show this help message and exit
  --add-repository URL  the URL of an additional repository that is required